packages/ttpassgen/PKGBUILD

```diff
@@ -0,0 +1,31 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=TTPassGen
+pkgver=133.a06d99d
+pkgrel=1
+pkgdesc='is a highly flexiable and scriptable password dictionary generator base on Python, you can easily use various rules to generate the desired combination of words.'
+arch=('any')
+groups=('blackarch' 'blackarch-automation' 'blackarch-cracker')
+url='https://github.com/tp7309/TTPassGen'
+license=('Apache')
+depends=('python' 'python-tqdm' 'python-click')
+makedepends=('git' 'python-setuptools')
+source=("git+https://github.com/tp7309/TTPassGen.git")
```

Comment:
Use `$_pkgname` here. No var in url but var in source is ok.

Reply:
Got it. Thank you for clarification.